File: lesunny/024_이상한_문자_만들기.java
import java.lang.*;

class Solution {
    public String solution(String s) {
        String answer = "";
        
        String[] arr = s.split(" ",-1); //https://elmkm.tistory.com/20
        
        for(int i=0;i<arr.length; i++){
            arr[i].trim();
            for(int j=0; j<arr[i].length();j++){
                if(j%2==0)
                    answer += Character.toString(Character.toUpperCase(arr[i].charAt(j)));
                else
                    answer += Character.toString(Character.toLowerCase(arr[i].charAt(j)));
            }
            answer+=" ";
        }
        
        return answer.substring(0, answer.length()-1);//answer.length()-1);
    }
}

File: JIWON/024 이상한 문자 만들기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s) {
    string answer = "";
    int count = 0;
    
    for(int i=0;i<s.size();i++){
        if(s[i] == ' ') {
            count = 1;
        }
        
        if(count % 2 == 0){
            answer += toupper(s[i]);    
        } else{
            answer += tolower(s[i]);
        }
        count++;
    }
    
    return answer;
}
